src/app/modules/entry-live/widgets/widget-base.ts

```diff
@@ -20,70 +20,70 @@ export abstract class WidgetBase<T = any> implements OnDestroy {
   protected _data = new BehaviorSubject<T>(null);
   protected _state = new BehaviorSubject<WidgetState>({ polling: false, activated: false, isBusy: false });
   protected _activationArgs: WidgetsActivationArgs;
-  
+
```

Comment:
there are no other changes in this file except line tabs removal, could you please revert changes for this file then